### ISSUE TITLE
ModuleNotFoundError: No module named 'conda'

### ISSUE THREAD

**Issue by veryscaryone** [pinokiocomputer/pinokio#237]
The app was already installed on my pc and used many times before. This time when I opened the program I wanted to use I got an install pop up. Clicked install and got this error message. I couldn't figure out why. I even uninstalled, restarted pc, reinstalled pinokio. Then I remembered I was having a problem with a plugin for davinci resolve. I had to add a new path, PYTHONHOME, to global env to get the plugin to work. I deleted the new path and pinokio worked. 

Is there anyway to get pinokio to work without having to delete the PYTHONHOME path?

ModuleNotFoundError: No module named 'conda'
Traceback (most recent call last):
  File "C:\pinokio\bin\miniconda\Scripts\conda-script.py", line 11, in <module>
    from conda.cli import main

**Comment by tomasris** [comment]
Yes, the program behaves strangely. I can confirm that after uninstalling the program, it does not want to install again conda, git and zip and its other applications, without which the program does not start. Only after cleaning the register, and all the files on the disc manually reinstalled. This is how I have it start with these windows. I understand this is the activation of conda.

https://github.com/user-attachments/assets/f70de3fc-d0a6-479f-93c0-bb938d5ac988

**Comment by eljabbaryhicham** [comment]
same problem, any fix?
